File: phoenix_books/priv/repo/seeds.exs
# Script for populating the database. You can run it as:
#
#     mix run priv/repo/seeds.exs
#
# Inside the script, you can read and write to any of your
# repositories directly:
#
#     PhoenixBooks.Repo.insert!(%PhoenixBooks.SomeModel{})
#
# We recommend using the bang functions (`insert!`, `update!`
# and so on) as they will fail if something goes wrong.

PhoenixBooks.Repo.insert!(%PhoenixBooks.Book{title: "Ready Player One", author: "Ernest Cline"})
PhoenixBooks.Repo.insert!(%PhoenixBooks.Book{title: "Rails for dummies", author: "Alex Kovshovik"})
PhoenixBooks.Repo.insert!(%PhoenixBooks.Book{title: "Benefits mode easy", author: "Jim Scott"})

File: performance/rails_books/db/seeds.rb
# This file should contain all the record creation needed to seed the database with its default values.
# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
#
# Examples:
#
#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
#   Character.create(name: 'Luke', movie: movies.first)

10.times do
	Book.create(title: "Ready Player One", author: "Ernest Cline")
	Book.create(title: "Rails for dummies", author: "Alex Kovshovik")
	Book.create(title: "Benefits mode easy", author: "Jim Scott")
end
